### PR TITLE
Remove password aging customized value to set default

### PR DESCRIPTION
**Pull request by chalamalasetty**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x ] The toolchain/worker package manifests are up-to-date
- [x ] Any updated packages successfully build (or no packages were changed)
- [x ] All package sources are available
- [x ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
 Currently 90 days older Mariner VHD images are failing with provisioning due to PAM complaining about authentication token no longer valid and password aging. Increasing maximum number of days a password can be used to 99999 which is default by removing customized value. Most of Linux distros have default value 99999 used for password aging. 

###### Change Log  

- Change 
-   Remove password aging customized value to set default.

###### Does this affect the toolchain?  

**YES**
NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/...

###### Test Methodology

- Built changes locally and verified the fix to confirm password aging set to default.

### CODE REVIEW

File: SPECS/shadow-utils/shadow-utils.spec [hunk 1]
```diff
@@ -36,27 +36,27 @@ in a secure way.
 sed -i 's/groups$(EXEEXT) //' src/Makefile.in
 find man -name Makefile.in -exec sed -i 's/groups\.1 / /' {} \;
 sed -i -e 's@#ENCRYPT_METHOD DES@ENCRYPT_METHOD SHA512@' \
-    -e 's@/var/spool/mail@/var/mail@' etc/login.defs
+    -e 's@%{_var}/spool/mail@%{_var}/mail@' etc/login.defs
```
Comment (anphel31) [inline SPECS/shadow-utils/shadow-utils.spec:39]: I believe these macros should not be used for package directory paths in the build chroot (linter needs to change). They should only be used for paths on the host system (such as in the %files section).

**Comment by rlmenge** [comment]
I recently merged a PR that might help with some of these concerns. Users can now set "-1" in the PasswordExpireDays field so their passwords never expire

https://github.com/microsoft/CBL-Mariner/pull/419

**Review by niontive — APPROVED** [review]

File: SPECS/shadow-utils/shadow-utils.spec (hunk 1 above)
Comment (thcrain-msft) [reply SPECS/shadow-utils/shadow-utils.spec:39]: Yes- changes to the linter are coming to make it less aggressive in this regard.

**Comment by PawelWMS** [comment]
Please indicate how you've tested the change in the `Test Methodology` section and move the description into the `Summary` part. We're trying to keep the PRs clean and uniform.

File: SPECS/shadow-utils/shadow-utils.spec (hunk 1 above)
Comment (PawelWMS) [reply SPECS/shadow-utils/shadow-utils.spec:39]: +1. And to be clear: the changes in the `%build`, `%install` and `%file` sections look good. In here we can leave the second macro (right after the second `@` symbol).

Comment (chalamalasetty) [reply SPECS/shadow-utils/shadow-utils.spec]: Updated PR. 
Thomas can you help removing var macro check for linter.

**Review by jslobodzian — APPROVED** [review]

**Review by anphel31 — APPROVED** [review]

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**